pymc/tests/test_sampling.py

```diff
@@ -625,23 +625,30 @@ def test_normal_scalar(self):
                 chains=nchains,
             )
 
+        random_state = self.get_random_state()
         with model:
             # test list input
             ppc0 = pm.sample_posterior_predictive(
-                10 * [model.initial_point()], return_inferencedata=False
+                10 * [model.initial_point()], return_inferencedata=False, random_seed=random_state
             )
-            # # deprecated argument is not introduced to fast version [2019/08/20:rpg]
-            ppc = pm.sample_posterior_predictive(trace, var_names=["a"], return_inferencedata=False)
+            assert "a" in ppc0
+            assert len(ppc0["a"][0]) == 10
             # test empty ppc
-            ppc = pm.sample_posterior_predictive(trace, var_names=[], return_inferencedata=False)
+            ppc = pm.sample_posterior_predictive(
+                trace, var_names=[], return_inferencedata=False, random_seed=random_state
+            )
             assert len(ppc) == 0
 
             # test keep_size parameter
-            ppc = pm.sample_posterior_predictive(trace, return_inferencedata=False)
+            ppc = pm.sample_posterior_predictive(
+                trace, return_inferencedata=False, random_seed=random_state
+            )
             assert ppc["a"].shape == (nchains, ndraws)
 
             # test default case
-            idata_ppc = pm.sample_posterior_predictive(trace, var_names=["a"])
+            idata_ppc = pm.sample_posterior_predictive(
+                trace, var_names=["a"], random_seed=random_state
+            )
```

Comment:
Only this test require a seed, the others are not checking anything that depends on the draws. Although passing a seed doesn't hurt, makes reading the git history less straightforward (unless it was a commit about adding random_seed to all tests, and not just this one in particular)

Reply:
I see, removed the seed from all the others.